src/App.tsx

```diff
@@ -1,7 +1,17 @@
+import { Route, Routes } from 'react-router-dom';
 import Hello from '~/components/Hello';
+import { DevPlayground } from './components/DevPlayground/DevPlayground';
+import { Navigation } from './components/Navigation/Navigation';
 
 function App() {
-  return <Hello />;
+  return (
+    <Routes>
+      <Route path='/' element={<Navigation />}>
```

Comment:
Dodałem nawigację z `<Outlet />` a pod nią podpięte pozostałe routy

Reply:
I would change the name of Navigation component to Layout or sth like that as it handles also routes